src/gluonnlp/data/utils.py

```diff
@@ -338,21 +360,26 @@ def _load_pretrained_vocab(name, root, cls=None):
             pass
         else:
             raise e
-
     if check_sha1(file_path, sha1_hash):
-        return _load_vocab_file(file_path, cls)
+        return _load_vocab_file(file_path, cls, special_tokens)
     else:
         raise ValueError('Downloaded file has different hash. Please try again.')
 
 
-def _load_vocab_file(file_path, cls):
+def _load_vocab_file(file_path, cls, special_tokens):
```

Comment:
should we replace special_tokens with kwargs so that we can easily customize vocab creation?

Reply:
Sure.